src/App/Components/CreateArticle/CreateArticle.js

```diff
@@ -57,23 +60,44 @@ export class CreateArticle extends Component {
             };
             this.setState({
               initialState: patchableArticle,
-              article: patchableArticle
+              article: patchableArticle,
+              isChanged: false
             });
           });
-        } else {
+        } else if (isChanged === true) {
           autosaveArticle(article);
+          this.setState({ isChanged: false });
         }
       }
-    }, 15000);
+    }, 6000);
+  }
+
+  componentWillMount() {
```

Comment:
@truestbyheart any reason why you're adding this to  `componentWillMount` instead of `componentDidMount`?

Reply:
I am using this life cycle to run some initial preparations before the component mounts...for instance, I am checking if the article to be updated is a draft so I can make a proper request to the backend